scripts/datasets/check_bcss.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from typing import Optional, List
+
+import vigra
+
+from torch_em.util.debug import check_loader
+from torch_em.data.datasets import get_bcss_loader
+
+
+# set this path to where you have downloaded the bcss data
+BCSS_ROOT = "/scratch/projects/nim00007/data/bcss/"
+
+
+class BCSSLabelTrafo():
+    def __init__(
+            self,
+            label_choices: Optional[List[int]] = None
+    ):
+        self.label_choices = label_choices
+
+    def __call__(
+            self,
+            labels: np.ndarray
+    ) -> np.ndarray:
+        """Returns the transformed labels (use-case for SAM)
```

Comment:
For the same use-case we should use connected components (`skimage.measure.label`) instead of just relabeling.
This way we can separate disconnected pieces, and have an instance segmentation, which will be better for SAM.
(Since this is just the sample script it's not important to update this here, but please keep it in mind for SAM.)